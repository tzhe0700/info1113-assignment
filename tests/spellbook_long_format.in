ADD SPELLBOOK spellbooks.csv 111111
SPELLBOOK 111111 LONG
EXIT
