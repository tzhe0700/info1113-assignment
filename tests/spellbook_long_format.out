user: Successfully added: Advanced Potion Making (Libatius Borage).

user: 111111: Advanced Potion Making (Libatius Borage, Curse)
Currently available.

user: Ending Archive process.

